src/inmanta/data/__init__.py

```diff
@@ -3346,6 +3348,9 @@ class Compile(BaseDocument):
 
     compile_data: Optional[JsonType] = None
 
+    partial: bool = False
+    removed_resource_sets: Optional[List[str]] = None
```

Comment:
Out of curiosity, did you try to use the built-in `list`? I hadn't considered this before but it wouldn't surprise me if our data framework (and our typed methods) isn't compatible with that. It would be a good thing to be aware of.

Reply:
I did not. Should I try it? :p

Reply:
If you feel like it, otherwise I'll give it a go myself sometime.

Reply:
mypy doesn't complain. I'll keep `list`

Reply:
Ho, that we'll see when I push then